etna/models/catboost.py

```diff
@@ -161,22 +166,15 @@ def __init__(
             not affect the training.
             During the training one main thread and one thread for each GPU are used.
         """
-        self.iterations = iterations
```

Comment:
Keep it, it is necessary for `__repr__` method.

Reply:
Why do we have it only for CatBoostModelMultiSegment but not for CatBoostModelPerSegment?

Reply:
We have it in all models except for CatBoost and SARIMAX. Then we should add it.

Reply:
Add it to `CatBoostModelPerSegment` please 🙂

Reply:
Done